packages/core/useDraggable/index.ts

```diff
@@ -96,8 +96,8 @@ export function useDraggable(target: MaybeComputedRef<HTMLElement | SVGElement |
       return
     const rect = resolveUnref(target)!.getBoundingClientRect()
     const pos = {
-      x: e.pageX - rect.left,
-      y: e.pageY - rect.top,
+      x: e.clientX - rect.left,
+      y: e.clientY - rect.top,
```

Comment:
One interesting tidbit is that this pretty much assumes "position: fixed". That's also the only thing that can be implemented in a sane manner, as far as I can tell.